daemon/algod/api/server/v2/utils.go

```diff
@@ -86,6 +86,13 @@ func byteOrNil(data []byte) *[]byte {
 	return &data
 }
 
+func nilToZero(numPtr *uint64) uint64 {
```

Comment:
I thought I had seen a function like this before.  Am I misremembering?

Reply:
I grep for similar pattern, and it turned out only `maxRound`?
```
❯ grep -r "*uint64) uint64 {"
./cmd/goal/account.go:func maxRound(current uint64, next *uint64) uint64 {
./daemon/algod/api/server/v2/utils.go:func nilToZero(numPtr *uint64) uint64 {
```

Reply:
Thanks for looking. I believe I was thinking about the functions above this, but they are for going in the other direction - taking things and converting zero values to nil.  This is good to have here.